templates/blender/contact-us.html

```diff
@@ -0,0 +1,15 @@
+{% extends "blender/_base-blender.html" %}
+
+{% block title %}Contact us | Blender{% endblock %}
+
+{% block content %}
+
+{% with h1="Let’s get in touch",
+  intro_text="Blender has partnered with Canonical to offer enterprise-grade support for the Blender LTS application suite. We can help you unleash more capabilities and supply the right level of support you need to have the best, hassle-free experience.",
+  formid="4052",
+  lpId="",
+  returnURL="" %}
```

Comment:
This needs a thank you page, the URL of which should be passed here, at the moment submitting the form looks like it just clears the form, there's no positive feedback that lets you know the form was successfully submitted. Take a look at https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/gov/thank-you.html, at first glance it looks like you could use almost the same markup.